### PR TITLE
Export Shared Notes to PDF/DOC/ODT

### PR DESCRIPTION
**Pull request by gustavotrott**
This PR will enable Shared Notes to be exportable to PDF, DOC and ODT.

image

This changes improve the office-convert scripts (#11717):
- Support inform output format as third param (supporting DOC / ODT);
- Includes param `--writer` when the input extension is `.html` (as Etherpad exports requires).

### To enable PDF / DOC /ODT export in Etherpad
A new script is available for this: `bbb-libreoffice/assets/etherpad-export.sh`.

1. Copy `bbb-libreoffice/assets/etherpad-export.sh` to `/usr/share/etherpad-lite/etherpad-export.sh`;
2. Edit `/usr/share/etherpad-lite/settings.json`;
3. Set `soffice` config to  `/usr/share/etherpad-lite/etherpad-export.sh`.
image

### CODE REVIEW

**Comment by pedrobmarin** [comment]
Hi @gustavotrott , thanks for this!

The MS Word format is usually not available too, is it using this docker endpoint too?

**Comment by gustavotrott** [comment]
> 
> 
> Hi @gustavotrott , thanks for this!
> 
> The MS Word format is usually not available too, is it using this docker endpoint too?

Thank you for letting me know  about that!
I just updated the PR to include support for .DOC exportation too.

**Comment by antobinary** [comment]
@gustavotrott Anything left until this can be tested/merged?

**Comment by gustavotrott** [comment]
> 
> 
> @gustavotrott Anything left until this can be tested/merged?

Everything is ok now.
I just changed the bbb-libreoffice installs to copy the `etherpad-export.sh` to `/usr/share/bbb-libreoffice-conversion/etherpad-export.sh`.
So the config of Etherpad has to be changed with this path for `soffice`.

**Comment by antobinary** [comment]
Awesome! Does this PR require any packaging changes?

**Comment by gustavotrott** [comment]
It should run again `bbb-libreoffice/install-local.sh` or `bbb-libreoffice/install-remote.sh`.

File: bbb-libreoffice/assets/convert-local.sh [hunk 1]
```diff
@@ -1,19 +1,31 @@
...
+convertToParam="--convert-to $convertTo"
+
+#If output is html, include param --writer to avoid blank page
+if [ ${1: -5} == ".html" ]
+then
+	convertToParam="$convertToParam --writer"
+fi
+
 mkdir -p "/tmp/bbb-libreoffice-conversion/"
+chmod 777 "/tmp/bbb-libreoffice-conversion/"
```
Comment (basisbit) [inline bbb-libreoffice/assets/convert-local.sh:24]: Why was 777 chosen here? Please change this asap.

**Pull request merged**